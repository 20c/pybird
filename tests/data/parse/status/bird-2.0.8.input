0001 BIRD 2.0.8 ready.
1000-BIRD 2.0.8
1011-Router ID is 1.1.1.1
 Hostname is bird2-router
 Current server time is 2022-01-25 21:45:13.551
 Last reboot on 2022-01-22 09:50:55.973
 Last reconfiguration on 2022-01-24 22:14:37.341
0013 Daemon is up and running
