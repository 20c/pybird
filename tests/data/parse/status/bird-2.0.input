0001 BIRD 2.0.7 ready.
1000-BIRD 2.0.7
1011-Router ID is 1.1.1.1
 Current server time is 2022-01-25 21:45:13.551
 Last reboot on 2022-01-22 09:50:55.973
 Last reconfiguration on 2022-01-24 22:14:37.341
0013 Daemon is up and running
