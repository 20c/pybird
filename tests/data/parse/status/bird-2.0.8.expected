{"version": "2.0.8", "router_id": "1.1.1.1", "hostname": "bird2-router", "last_reboot": "2022-01-22T09:50:55", "last_reconfiguration": "2022-01-24T22:14:37"}